querydsl-apt/src/test/java/com/querydsl/apt/DateExtensionsTest.java

```diff
@@ -34,7 +34,7 @@
     private static final String packagePath = "src/test/java/com/querydsl/apt/";
 
     @Test
-    public void Handles_Date_Extensions_Correctly() throws IOException, InterruptedException {
+    public void handles_Date_Extensions_Correctly() throws IOException, InterruptedException {
```

Comment:
I don't quite see what you mean with `camelCase_variant_in_lower_case_snake_case`.
But I think that `camelCase` is fine.